routes/celebrities.js

```diff
@@ -0,0 +1,64 @@
+const router = require('express').Router();
+const Celebrity = require('../models/celebrity');
+
+router.get('/celebrities', (req, res, next) => {
+  Celebrity.find()
+    .then(celebrities => {
+      res.render('celebrities/index', { celebrityList: celebrities });
+    }).catch(err => {
+      next(err);
+    });
+});
+
+router.post('/celebrities', (req, res) => {
+  //In this case it also works to directly pass the req.body object. Is there a reason not do it like this?
+  Celebrity.create(req.body)
+    .then(() => {
+    res.redirect('/celebrities');
+  }).catch(err => {
+    console.log(err);
+    res.render('celebrities/new');
+  });
+});
+
+router.get('/celebrities/new', (req, res) => {
```

Comment:
If the base path in "app.use" is "/celebrities", then the path defined here would be "/new".